pages/Documents/ConversationalAI/IntentManager/key-terms-concepts.md

```diff
@@ -145,34 +145,48 @@ A meta intent is a wrapper that can contain many other standard intents. This fu
 
 ### Entities
 
-Entities are keywords or expressions that represent groups of items. For example, an entity named `SIZE` might represent the following values: small, medium, and large.
+Entities are the things, and their attributes, referred to in consumer conversations. More formally, a named entity is a set of elements that is important to understanding raw text, but is not necessarily related to the *intent* of the raw text. In general, entities can be swapped while preserving the general intent of the text.
+
+Often, named entities are the nouns and adjectives in a sentence, but they can be almost any part of speech. LivePerson’s entity definitions are flexible, meaning entities are whatever you want to capture from your customer conversations. Some examples:
+
+<img style="width:600px" alt="Examples of entities in consumer messages" src="img/ConvoBuilder/im_entities_examples.jpg">
+
+In other words, entities enable artificial intelligence to understand text at a deeper level. Without entities, artificial intelligence would know someone wants to buy something, but wouldn’t know *what* they want to buy, or *where* they want it delivered, or *how* they want to pay. That’s why [named entity recognition](https://en.wikipedia.org/wiki/Named-entity_recognition), a natural language processing technique for automatically identifying entities in a sentence and classifying them into pre-defined categories, is so important.
```

Comment:
That's why named entity recognition, a natural language processing technique for automatically identifying entities in a sentence and classifying them into pre-defined categories, is crucial.

Reply:
Keeping Curiously Human language; entities aren't necessary to use, so "crucial" seems not quite right, although close. Thank you.